src/client/pages/Offer/Checkout/SignDisclaimer.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react'
```

Comment:
We refactored it to avoid passing all of "offerData"

We changed from "signMethod" to "checkoutMethod" in the props.

And updated to use the new locale hook.